README.md

```diff
@@ -378,6 +378,12 @@ const schema = {
 
 > Note that storing large dataURIs into form state might slow rendering.
 
+##### File widget input ref
+
+The included `FileWidget` exposes its `<input type="file"></input>` via `inputRef`.
```

Comment:
Nit: *The included `FileWidget` exposes a reference to the `<input type="file" />` element node as a `inputRef component property.*